src/bpftrace.cpp

```diff
@@ -45,7 +45,7 @@
 
 namespace bpftrace {
 
-DebugLevel bt_debug = DebugLevel::kNone;
+std::set<DebugStage> bt_debug = {};
```

Comment:
Maybe this should go into the config class? WDYT?

Reply:
I'm not a big fan of that. It's not something that affects bpftrace behaviour, it's just enabling more prints. I'm not aware of any tool which would allow passing the verbosity/debug options via a config.

Reply:
Yeah, I guess I wasn't thinking about it being set from inside the script itself, which I also don't like. It just seemed like some level of "configuration".